modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -654,6 +655,35 @@ enum { CAP_PROP_IMAGES_BASE = 18000,
 
 //! @} Images
 
+/** @name OBSENSOR (for Orbbec 3D-Sensor device/module )
+    @{
+*/
+//! OBSENSOR data given from iamge generator
+enum { 
```

Comment:
> `enum {`

Avoid using of anonymous enums. Add name for them and put into documentation information about where it should be used.